CONTRIBUTING.md

```diff
@@ -119,7 +119,7 @@ This repository and your GitHub clone will be linked to your local clone to bein
 3. Add this repository as a 2nd `remote` so you can always pull the latest changes.
    Here is the command to add this repository with the "wo-ist-markt" alias:
 
-   ``` bash
```

Comment:
This change actually undoes the indentation of the code block. Please have a look at the [preview](https://github.com/petedreizehn/wo-ist-markt.github.io/blob/contrib-guide-fix/CONTRIBUTING.md#basic-setup).

Please remove the change. Feel free to `git commit --amend` and `git push --force origin contrib-guide-fix` your branch. - If this is not possible just let me know. Than I can do the cleanup.

Reply:
This is true, but it also broke the following formatting.
I aligned the code block to be intended with the other ones.